apps/openmw/mwworld/worldimp.cpp

```diff
@@ -1352,13 +1352,37 @@ namespace MWWorld
 
     void World::fixPosition(const Ptr &actor)
```

Comment:
Arguably it's unnecessary to have an argument for this function since it can only be used for the player.